src/Page/Options.elm

```diff
@@ -111,126 +111,64 @@ update navKey msg model =
 view : Model -> Html Msg
 view model =
     Search.view { toRoute = Route.Options, categoryName = "options" }
-        "Search NixOS options"
+        [ text "Search more than "
+        , strong [] [ text "10 000 options" ]
+        ]
         model
         viewSuccess
+        viewBuckets
         SearchMsg
 
 
+viewBuckets :
+    Maybe String
+    -> Search.SearchResult ResultItemSource ResultAggregations
+    -> List (Html Msg)
+viewBuckets _ _ =
+    []
```

Comment:
I think putting `(\_ _ -> [])` directly on the line 119 would make it more obvious that we don't expect anything here in this case and that this is intentional.

Reply:
I've added this as a placeholder. Also you wouldn't know what that function should return if we would make it anonymous.